### PR TITLE
Add OpenRCT2Launcher v0.0.7

### PR DESCRIPTION
**Pull request by jkap**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [ ] Named the cask according to the [token reference].
  - Name differs from the generated token, as there is already an `openrct2` formula and the repo and colloquial name for this app is OpenRCT2 Launcher.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/openrct2-launcher.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'openrct2-launcher' do
```
Comment (vitorgalvao) [inline Casks/openrct2-launcher.rb]: Seems to be a candidate for the [`-app` suffix](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md#token-overlap-with-a-formula).

**Comment by vitorgalvao** [comment]
Unrelated to the PR, I like the illustration style of your avatar. I’m guessing it was a commission (since a cursory glance at your website doesn’t mention illustration). I’d like to see more. Can you point me to the artist?

**Comment by jkap** [comment]
Good guess, it was definitely a commission! Had it done a couple years ago by https://mobile.twitter.com/Kaibuzetta!

File: Casks/openrct2-launcher.rb (hunk 1 above)
Comment (jkap) [reply Casks/openrct2-launcher.rb]: yeah, i thought that might be the case but i wasn't 100% sure. made the change in 9ae6469a58d27b547306fd4e2eeac85e19760e84

**Comment by vitorgalvao** [comment]
@jkap Thank you!

**Pull request merged**